Assets/Engine/Inventory/UI/ContainerUi.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using UnityEngine;
 using UnityEngine.UI;
+using SS3D.Content.Systems.Examine.Examinables;
```

Comment:
Again, please remove the use of Content.

Reply:
Done